### PR TITLE
Updated the quality of Title2 (Robocop.mp3)

### PR DESCRIPTION
**Pull request by actioninja**
NOT SHITPOST REAL PR I SWEAR

## About The Pull Request

For some reason the version of Robocop.mp3 in the game files was ridiculously low quality.
Now uses the version found [here.](https://www.youtube.com/watch?v=vHo7npmGcHU) ~~Ripped directly from the video as the description download is lower quality than the actual video for some bizarre reason.~~ Now uses a version uploaded to the internet archive which may actually be the original upload, really not sure. In either case it's a little smaller while sounding identical to my ears.
Horribly compressed version has been uploaded [here](https://www.youtube.com/watch?v=dCPWE4WexM8) for reference.

## Why It's Good For The Game

Listen to that beautiful robocop theme but in higher quality now.

## Changelog
:cl:
soundadd: Replaced Title2.ogg with a much higher quality version.
/:cl:

ＤＥＨＵＭＡＮＩＺＥ ＹＯＵＲＳＥＬＦ ＡＮＤ ＦＡＣＥ ＴＯ ＢＬＯＯＤＳＨＥＤ

### CODE REVIEW

**Comment by plapatin** [comment]
the compressed part of title2 is the best part

**Comment by actioninja** [comment]
The fact that's genuinely a good song is the best part. It sounds like complete dogshit in the game files.

**Comment by actioninja** [comment]
Fuck you appveyor it is literally a sound file change and nothing else you piece of shit

**Comment by actioninja** [comment]
oh shit if you edit that file it tags mso oh god oh fuck

**Comment by Barhandar** [comment]
ONE MEGABYTE MORE TO DOWNLOAD FOR CLIENTS

**Comment by actioninja** [comment]
> ONE MEGABYTE MORE TO DOWNLOAD FOR CLIENTS

OOOOOH NOOOOOO
Title3 is still over double the size.

**Comment by PKPenguin321** [comment]
Don't say the R WORD YOU'LL GET US ALL KILLED

**Comment by actioninja** [comment]
> Don't say the R WORD YOU'LL GET US ALL KILLED

OH SHIT

**Review by ShizCalev — DISMISSED** [review]

**Review by ShizCalev — APPROVED** [review]

**Comment by daklaj** [comment]
human ear cant hear more than 1kb/s anyway

File: code/controllers/subsystem.dm [hunk 1]
```diff
@@ -215,3 +215,4 @@
 		if ("queued_priority") //editing this breaks things.
 			return 0
 	. = ..()
+
```
Comment (mattroks101) [inline code/controllers/subsystem.dm]: 🤔

File: code/controllers/subsystem.dm [hunk 2]
```diff
@@ -215,3 +215,4 @@
 		if ("queued_priority") //editing this breaks things.
 			return 0
 	. = ..()
```
Comment (ShizCalev) [inline code/controllers/subsystem.dm]: return ..() while you're here

**Comment by ShizCalev** [comment]
Tbh I like the compression on the old version as it seems to add to the sort of retro appeal of the game, but having a higher quality file is definitely much butter.

**Comment by optimumtact** [comment]
did you downconvert it to mono?

**Comment by ShizCalev** [comment]
@optimumtact No, it's in stereo, as are all the other `title*.ogg` files.

**Comment by optimumtact** [comment]
byond only plays mono sound

**Comment by ShizCalev** [comment]
@optimumtact Waveform analysis of audio output on the title screen suggests otherwise? (Example is title1.ogg)
image

Also just listening to title1.ogg ingame, it's definitely playing in proper stereo.

**Comment by actioninja** [comment]
It's positional audio that gets crushed to mono, ambient audio and music is fully stereo.

**Comment by PKPenguin321** [comment]
Fwiw the title songs *could* be converted to mono if we care about filesize, but yeah it doesn't matter much

**Pull request merged**